src/lwc/geFormWidget/geFormWidget.js

```diff
@@ -12,13 +12,15 @@ import DATA_IMPORT_PAYMENT_STATUS from '@salesforce/schema/DataImport__c.Payment
 import DATA_IMPORT_PARENT_BATCH_LOOKUP from '@salesforce/schema/DataImport__c.NPSP_Data_Import_Batch__c';
 import DATA_IMPORT_ID from '@salesforce/schema/DataImport__c.Id';
 
-const PAYMENT_SCHEDULER_WIDGET = 'geFormWidgetPaymentScheduler';
```

Comment:
So this row actually just bumped down here, but maybe it would actually better to clear out the references to payment scheduler now that I'm thinking about it? There are a few!